blocks/init/src/Blocks/components/layout/layout.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Layout component - Grid Columns grid.
```

Comment:
Now I'm unsure about my comment in the storybook readme.

What is this layout component actually doing?

Just providing the CSS grid wrapper? Wouldn't then it be better for it to be named `grid` or `layout-grid`, instead of just layout?

I think the core (or just wp.com) have a Layout Grid block:

<img width="664" alt="image" src="https://user-images.githubusercontent.com/8638515/172107685-f0839a73-d261-4918-b886-fbc8314cd1d4.png">

Layout implies a specific structure

Reply:
the idea here is to have any kind of layouts here not just grid like. So this is why I just called it layout. I will update the readme